<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Viju" ignored="0" name="Default test" tests="1" failures="1" timestamp="2022-06-01T09:40:23 IST" time="35.123" errors="0">
  <testcase name="HoldingList" time="13.038" classname="KiteHomeTest.Fail_Test_HoldingList">
    <failure type="org.openqa.selenium.NoSuchElementException" message="Cannot locate option with text: Mutual Funds
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: &amp;apos;4.2.0&amp;apos;, revision: &amp;apos;86eb611648&amp;apos;
System info: host: &amp;apos;VIJU&amp;apos;, ip: &amp;apos;192.168.43.248&amp;apos;, os.name: &amp;apos;Windows 8&amp;apos;, os.arch: &amp;apos;x86&amp;apos;, os.version: &amp;apos;6.2&amp;apos;, java.version: &amp;apos;1.8.0_321&amp;apos;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate option with text: Mutual Funds
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.2.0', revision: '86eb611648'
System info: host: 'VIJU', ip: '192.168.43.248', os.name: 'Windows 8', os.arch: 'x86', os.version: '6.2', java.version: '1.8.0_321'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.selectByVisibleText(Select.java:151)
at libraryfiles.UtilityClass.Fail_list(UtilityClass.java:41)
at KiteHomeTest.Fail_Test_HoldingList.HoldingList(Fail_Test_HoldingList.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- HoldingList -->
</testsuite> <!-- Default test -->
